input/intro-notes/StructureDefinition-au-core-location-intro.md

```diff
@@ -6,16 +6,6 @@ The following are supported usage scenarios for this profile:
 - Read information about a location referenced by another resource
```

Comment:
Commenting here to get an inline comment. 

Missing a change to add a sentence to intro, above the Usage Scenarios, to link to the Comparison page that is present in other profiles pages.

Reply:
fixed